src/Language/PureScript/CodeGen/Externs.hs

```diff
@@ -38,11 +38,24 @@ import Language.PureScript.Types
 moduleToPs :: Module -> Environment -> String
 moduleToPs (Module _ _ _ Nothing) _ = error "Module exports were not elaborated in moduleToPs"
 moduleToPs (Module _ moduleName ds (Just exts)) env = intercalate "\n" . execWriter $ do
-  tell [ "module " ++ runModuleName moduleName ++ " where"]
+  let exps = listExports exts
+  tell ["module " ++ runModuleName moduleName ++ (if null exps then "" else " (" ++ exps ++ ")") ++ " where"]
```

Comment:
Ah! In that case we can probably ignore it, if it's not possible to produce a module with `()` exports manually I don't think we'll ever make a module with `null exps` except in situations where the module is empty anyway.

I was thinking it might be handy for a module that only exports instances, but that shouldn't be a thing anyway, as it means the instances are all orphans if no types or classes are exported along with them.